src/test/java/core/basesyntax/service/impl/FruitServiceImplTest.java

```diff
@@ -0,0 +1,57 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.Operation;
+import core.basesyntax.model.TransactionDto;
+import core.basesyntax.service.FruitService;
+import core.basesyntax.strategy.BalanceStrategy;
+import core.basesyntax.strategy.OperationStrategy;
+import core.basesyntax.strategy.PurchaseStrategy;
+import core.basesyntax.strategy.ReturnStrategy;
+import core.basesyntax.strategy.SupplyStrategy;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class FruitServiceImplTest {
+    @Test
+    public void correctStorage_Ok() {
+        List<TransactionDto> transactionDtoList = List.of(
+                new TransactionDto(Operation.BALANCE, new Fruit("banana"), 20),
+                new TransactionDto(Operation.BALANCE, new Fruit("apple"), 100),
+                new TransactionDto(Operation.PURCHASE, new Fruit("banana"), 13),
+                new TransactionDto(Operation.RETURN, new Fruit("apple"), 10),
+                new TransactionDto(Operation.SUPPLY, new Fruit("banana"), 20));
+
+        FruitService fruitService = new FruitServiceImpl(operationStrategyMap());
```

Comment:
move creating of this service to BeforeClass method. YOu don't need a few of them for each test